std/string.d

```diff
@@ -7394,16 +7394,27 @@ Params:
 Returns:
     arr retyped as an array of chars, wchars, or dchars
 
+Throws:
+    In debug mode `AssertError`, when the result is not a well-formed UTF string.
+
 See_Also: $(LREF representation)
 */
-auto assumeUTF(T)(T[] arr) pure
+auto assumeUTF(T)(T[] arr) pure nothrow @nogc
```

Comment:
Unless there's a compiler error, none of the attributes should be required. The way to make sure the attributes are inferred correctly is to add them to a unittest that calls assumeUTF.

Reply:
OK. Moved the attributes in the unittest.